src/coinbase/address.ts

```diff
@@ -295,6 +297,31 @@ export class Address {
     );
     return new FaucetTransaction(response.data);
   }
+  /**
+   * Returns the reputation of the Address.
+   *
+   * @returns The reputation of the Address.
+   * @throws {Error} if the API request to get the Address reputation fails.
+   * @throws {Error} if the Address reputation is not available.
+   */
+  public async reputation(): Promise<AddressReputation> {
+    const response = await Coinbase.apiClients.addressReputation!.getAddressReputation(
+      this.getNetworkId(),
+      this.getId(),
+    );
+
+    this._reputation = new AddressReputation(response.data);
+    return this._reputation;
+  }
+
+  /**
+   * Returns whether Address's reputation is risky.
+   *
+   * @returns {boolean} true if the Address's reputation is risky
+   */
+  public risky(): boolean {
+    return this._reputation?.risky ?? false;
```

Comment:
It doesn't seem correct to deem an address "not risky", just because we have yet to fetch it.

Since in JS, this would need to be async in order for it to properly lazily evaluate, I'd recommend omitting this function at the address level, and instead have it live on the address reputation.